services/venice-router/src/main/java/com/linkedin/venice/router/RouterServer.java

```diff
@@ -183,14 +183,18 @@ public static void main(String args[]) throws Exception {
     if (args.length != 1) {
       Utils.exit("USAGE: java -jar venice-router-all.jar <router_config_file_path>");
     }
-    VeniceProperties props;
+
     try {
       String routerConfigFilePath = args[0];
-      props = Utils.parseProperties(routerConfigFilePath);
+      run(routerConfigFilePath, false);
```

Comment:
It seems we'll never run forever?  I'm trying to understand the use of the parameter to run.

Reply:
Great catch !

Actually the service worked anyway because there are some other non deamon threads running.

Fixed